main.go

```diff
@@ -241,14 +240,15 @@ func main() {
 
 	renewDeadline := time.Duration(leaderRenewSeconds) * time.Second
 	leaseDuration := time.Duration(int(1.2*float64(leaderRenewSeconds))) * time.Second
+	leaderRetryDuration := renewDeadline / 2
 
 	// Only attempt to grab leader lock once we have an API connection.
 	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
 		Scheme:                     scheme,
 		MetricsBindAddress:         metricsAddr,
 		Port:                       9443,
 		LeaderElection:             enableLeaderElection,
-		LeaderElectionID:           "storageos-api-manager-leader",
+		LeaderElectionID:           "storageos-api-manager-leader-asdekfww",
```

Comment:
"asdekfww" if we want to use a different name we could use something that made sense 😄

Reply:
This is a pattern i have seen, to give a unique ID for the lease. I know we don't necessary have to change this, but why not :)